wondrous-app/utils/enhancements/lazy.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+interface Load {
+  canLoad: boolean;
+  onLoad: () => void;
+}
+
+const lazy =
+  (importFn: () => Promise<any>, Fallback?: any) =>
+  ({ canLoad = true, onLoad, ...props }: Load) => {
```

Comment:
the onLoad and canLoad are just custom behavior that we are not utilizing right now right?

Reply:
correct